eng/pipelines/corefx-base.yml

```diff
@@ -0,0 +1,134 @@
+  preBuildSteps: []
+
+  # Optional
+  # Default: true
+  # true -- tests are built, but not run.
+  # false -- tests are built and run in the build machine.
+  skipTests: true
+
+  # Optional: Value to know if it should submit tests payloads to helix.
+  # Default: true
+  submitToHelix: true
+
+  # Optional: Value to know if it should test outerloop tests
+  # Default: false
+  testOuterloop: false
+
+  # Optional: Timeout for the job in minutes
+  # Default: 180
+  timeoutInMinutes: 180
+
+  # Optional: Value to specify if the build should run custom steps.
+  # Default: false
+  # When true, customBuildSteps should be populated with the steps to run.
+  useCustomBuildSteps: false
+
+jobs:
+  - template: ../common/templates/job/job.yml
+    parameters:
+      name: ${{ parameters.name }}
+      variables:
+        # OS Specific commands
+      - ${{ if eq(parameters.targetOS, 'Windows_NT') }}:
+        - name: _buildScript
+          value: build.cmd -ci
+        - name: _msbuildCommand
+          value: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
+      - ${{ if ne(parameters.targetOS, 'Windows_NT') }}:
+        - name: _buildScript
+          value: ${{ parameters.buildScriptPrefix }}./build.sh --ci
+        - name: _msbuildCommand
+          value: ${{ parameters.buildScriptPrefix }}./eng/common/msbuild.sh --warnaserror false --ci
+
+      - ${{ if eq(parameters.enableMicrobuild, 'true') }}:
+        - name: _TeamName
+          value: DotNetCore
+        - name: _SignType
+          value: true
+      - ${{ if eq(parameters.submitToHelix, 'true') }}:
+        - group: Helix Anonymous
+
+      enableMicrobuild: ${{ parameters.enableMicrobuild }}
+      displayName: ${{ parameters.displayName }}
+      timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+      pool:
+        ${{ insert }}: ${{ parameters.pool }}
+
+      ${{ if ne(parameters.strategy, '') }}:
+        strategy:
+          maxParallel: 99
+          ${{ insert }}: ${{ parameters.strategy }}
+
+      container: ${{ parameters.container }}
+
+      steps:
+        - ${{ parameters.preBuildSteps }}
+
+        - ${{ if eq(parameters.useCustomBuildSteps, 'false') }}:
+          - script: $(_buildScript)
+                  -includetests
+                  -framework $(framework)
+                  /p:ArchGroup=$(_architecture)
+                  /p:ConfigurationGroup=$(_configuration)
+                  /p:SkipTests=${{ parameters.skipTests }}
+                  /p:ArchiveTests=${{ parameters.submitToHelix }}
+                  /p:Outerloop=${{ parameters.testOuterloop }}
+                  ${{ parameters.buildExtraArguments }}
+            displayName: Build Sources and Tests
+
+        - ${{ if eq(parameters.useCustomBuildSteps, 'true') }}:
+          - ${{ parameters.customBuildSteps }}
+
+        - ${{ if eq(parameters.submitToHelix, 'true') }}:
+          - template: /eng/pipelines/helix.yml
```

Comment:
Is there no good way to get CoreFx working with https://github.com/dotnet/arcade/blob/d0d4619c658dd39555282fffcc9bb8a17117bc37/eng/common/templates/steps/helix-publish.yml?